### DISCUSSION TITLE
is there a way to set zoom out to gui.SceneWidget()?

### DISCUSSION THREAD

**Discussion by MeyerBuaharon** [isl-org/Open3D#3932] [Q&A] [answered]
I can't really find where to set my zoom on the widget.
This is how I set the look at:
```
widget = gui.SceneWidget()
widget.scene.camera.look_at([0, 0, 0], [1, 1, 1], [0, 0, 1])
# some code
widget.scene.add_geometry('frame', frame, mat)
```
Does anyone know how I can set up my zoom?

**Comment by hduonggithub** [comment, marked as answer]
The 2nd xyz (e.g., [1, 1, 1]) in the look_at is your camera position, changing this to bigger values means you move away from the target. It appears like zoom out. Vice versa is zoom in
